src/components/ErrorReport.vue

```diff
@@ -161,6 +162,10 @@ export default {
   overflow: hidden;
   white-space: nowrap;
   border-radius: 8px;
+  display: flex;
```

Comment:
I'm confused about this bit. Why are you setting it? It actually changes the internal formatting of the div

Reply:
![image](https://user-images.githubusercontent.com/4781841/154540130-c8baf52c-d388-4a8e-8976-40ced88c346d.png)